### ISSUE TITLE
Location page MVP (Janis)

### ISSUE THREAD

**Issue by briaguya** [cityofaustin/techstack#3297]
- On service and department pages, make sure we have the correct info:
   - Contact info as before
   - Location info:
     - If 1 location, show as alpha does now, but link to location page
     - If more than 1 location, show a list of links (janis design links?)
  - Location page (janis design links?)
   - make it match design

## Janis: location page (sprint 44)
- [x] Implementation matches mocks
  - [x] [desktop](https://xd.adobe.com/spec/d7653e44-403a-4620-4436-1484993d4622-a154/screen/62dce76d-7739-43a1-a958-9c2e122b74f8/Template-Places-MVP-Desktop)
  - [x] [Tablet](https://xd.adobe.com/spec/d7653e44-403a-4620-4436-1484993d4622-a154/screen/64aa180d-e5d8-470f-a53b-9fc814a37d57/Template-Places-MVP-Tablet)
  - [x] [Mobile](https://xd.adobe.com/spec/d7653e44-403a-4620-4436-1484993d4622-a154/screen/8b743758-76df-4678-8e92-0d8e7b66c31d/Template-Places-MVP-Mobile)
- [ ] If there is only a physical address (and no need for a specific mailing address), label just says "Address"
- [ ] Page preview implemented
- [ ] Translations: when page viewed in Spanish, the following microcopy is translated

| English copy        | Spanish copy           | 
| ------------- |-------------|
| *Contact*     | Contacto  |
| Phone     |  Teléfono     |
| Email address | Correo electrónico      |
| *Location*     | Ubicación  |
| Physical address     |  Dirección física     |
| Mailing address |  Dirección postal     |
| *Facility hours*     |  Horario del centro     |
| Standard hours |   Horario regular    |
| *Services offered*     |   Servicios ofrecidos    |
| Service information |  Información sobre el servicio     |
| See more hours     |  Vea otros horarios     |
| Close hours |   Cierre horarios    |
| *Getting here*     |  Cómo llegar     |
| Buses servicing this area   |   Autobuses del área   |

**Comment by desigonz** [comment]
[Janis mocks](https://xd.adobe.com/spec/d7653e44-403a-4620-4436-1484993d4622-a154/screen/64aa180d-e5d8-470f-a53b-9fc814a37d57/Option-1-tablet-icons-and-labels). Ignore the "accessibility features"—that's post-MVP
- [ ] Content needs to review the microcopy

**Comment by desigonz** [comment]
duplicate of #3400

**Comment by desigonz** [comment]
Actually maybe not? We can move acceptance criteria from #3400 during sprint planning tomorrow if that makes sense.

**Comment by desigonz** [comment]
By end of this sprint, we can see the Janis side!

**Comment by desigonz** [comment]
I sent out the microcopy for translation today, and will update the table above once they come in. @Niicck, if you need some placeholder translations before we can the official translation back, I'm happy to draft some up.

**Comment by Niicck** [comment]
@chasechenevert What's the source for this fancy little icon?
<img width="488" alt="Screen Shot 2019-11-19 at 2 37 39 PM" src="https://user-images.githubusercontent.com/14187277/69184306-38c7fc00-0ada-11ea-999a-4198bfa94dc3.png">

**Comment by desigonz** [comment]
@Niicck I updated the Spanish translations above

**Comment by chasechenevert** [comment]
> @chasechenevert What's the source for this fancy little icon?
> <img alt="Screen Shot 2019-11-19 at 2 37 39 PM" width="488" src="https://user-images.githubusercontent.com/14187277/69184306-38c7fc00-0ada-11ea-999a-4198bfa94dc3.png">

Nick and I discussed using the icon that we use for locations in the contact information component: https://alpha.austin.gov/en/housing-utilities/recycling-trash-and-compost/recycling-reuse/pickup-free-paint/

![Screen Shot 2019-11-22 at 9 15 14 AM](https://user-images.githubusercontent.com/7027270/69437298-a35e7f00-0d08-11ea-98bf-7d7395d1f70e.png)

Spec has been updated to reflect the change.

**Comment by Niicck** [comment]
Since there are no Joplin Location Pages yet, you can test it in storybook!
https://janis-location-storybook.netlify.com/?path=/story/pages-locationpage--basic

**Comment by desigonz** [comment]
Looks 😍!!! Assigning @chasechenevert to review that implementation matches mocks. Since we don't have it hooked up to Joplin yet, I'm leaving the last three acceptance criteria to test when we do integrate them.

**Comment by desigonz** [comment]
Since @chasechenevert is out for the rest of this sprint, assigning myself to QA.

**Comment by desigonz** [comment]
Awesome job @Niicck! Here's my feedback.

- [ ] Hyperlinks for phone numbers throughout this content type (`a href="tel:"`)—behavior should match the behavior on the [OPO page contact](https://alpha.austin.gov/police-oversight/) (see screenshot below)
- [ ] Hyperlink email address (`a href="mailto:"`)—behavior should match the behavior on the [OPO page contact](https://alpha.austin.gov/police-oversight/) (see screenshot below)
- [ ] Translations
  - [ ] Within hours, "Closed" should translate to "Cerrado"
  - [ ] Within hours, "Noon" should translate to "mediodía" (I'm noticing this isn't happening currently in [contacts](https://alpha.austin.gov/es/health-safety/healthcare-prevention/disease-prevention/get-immunized) :( Is this something you'd fix across the board, or only fix for this content type, and then we should create a new issue to fix elsewhere?)
  - [ ] "Mailing address" should translate to "Dirección postal"
- [ ] Also: in Joplin, entering phone number labels are supposed to be optional. You might have already done this, but I would expect that if there is no phone number label entered, then on Janis, only the phone number appears, without label


![image](https://user-images.githubusercontent.com/5839508/69660532-2738a400-1046-11ea-83f0-2d55905bd28b.png)

![image](https://user-images.githubusercontent.com/5839508/69660885-e8571e00-1046-11ea-94a5-63f8582b55ee.png)

**Comment by Niicck** [comment]
Just added those QA fixes. I needed to wait on the specific API implementation to fix a couple of those, so I moved them into https://github.com/cityofaustin/techstack/issues/3296
